stdlib/test/builtin/test_string.mojo

```diff
@@ -305,13 +305,21 @@ def test_ord():
     assert_equal(ord("🔥".as_string_slice()), 128293)
 
 
-def test_chr():
+def test_unicode():
```

Comment:
I think I would rather have 3 tests as now the test is covering `char`, `String.from_unicode` and `String.from_utf16`

Reply:
The undelying function is the same for all 3 `_shift_unicode_to_utf8` which I pulled out of the `chr` implementation, but yeah I can separate them